src/stylesheets/components/_navigation.scss

```diff
@@ -0,0 +1,49 @@
+.app-c-navigation {
+  padding-right: $govuk-spacing-scale-3;
+  padding-left: $govuk-spacing-scale-3;
+  background-color: $app-light-grey;
+  font-family: $govuk-font-stack;
+  @include font-smoothing;
+  @include govuk-bold-19;
+
+  @include mq($until: desktop) {
+    display: none;
+  }
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    & > li {
```

Comment:
are we set on doing & > type styling?

Reply:
Yes, as documented here: https://trello.com/c/mEN36uTR/30-should-we-have-class-names-for-all-elements-inside-components-or-use-descendant-selectors

Reply:
ql, 👍